src/util.py

```diff
@@ -260,12 +265,160 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def plotCLabels(da, contours, transform, ax, proj, clabel_locations=[], fontsize=12, whitebbox=False, horizontal=False):
```

Comment:
Just a suggestion: How about the following argument order?

`def plotCLabels(ax, contours, transform, proj, da, clabel_locations=[], fontsize=12, whitebbox=False, horizontal=False):`